libs/FreeImage/include/FreeImage.h

```diff
@@ -158,14 +158,25 @@ typedef int32_t LONG;
 typedef int64_t INT64;
 typedef uint64_t UINT64;
 #else
+// define portable types for 32-bit / 64-bit OS
+#include <inttypes.h>
+#define BOOL int32_t;
+#define BYTE uint8_t;
+#define WORD uint16_t;
+#define DWORD uint32_t;
+#define LONG int32_t;
+#define INT64 int64_t;
+#define UINT64 uint64_t;
+#endif // end __APPLE_CC__
+#else
 // MS is not C99 ISO compliant
-typedef long BOOL;
-typedef unsigned char BYTE;
-typedef unsigned short WORD;
-typedef unsigned long DWORD;
-typedef long LONG;
-typedef signed __int64 INT64;
-typedef unsigned __int64 UINT64;
+#define BOOL long;
+#define BYTE unsigned char;
+#define WORD unsigned short;
+#define DWORD unsigned long;
+#define LONG long;
+#define INT64 signed __int64;
+#define UINT64 unsigned __int64;
```

Comment:
anything with #define shouldn't have the ; at the end 
also from travis looks like the the apple check isn't working as its using the #define BYTE and not the typedef